sig-docs/annual-report-2022.md

```diff
@@ -105,3 +106,102 @@ Operational tasks in [sig-governance.md]:
+    * [Intro to Kubernetes Docs](https://www.youtube.com/watch?v=pprMgmNzDcw) – presented at KubeCon/CloudNativeCon NA, 2020
+    * [Kubernetes SIG Docs: A Deep Dive](https://www.youtube.com/watch?v=GDfcBF5et3Q) – presented at KubeCon/CloudNativeCon NA, 2021
+
+
+**Does the group have contributors from multiple companies/affiliations?**
+
+* Yes. We have a contributor base spread across [98 companies](https://k8s.devstats.cncf.io/d/8/company-statistics-by-repository-group?orgId=1&var-period=y&var-metric=contributions&var-repogroup_name=SIG%20Docs&var-repo_name=kubernetes%2Fkubernetes&var-companies=All&from=1640991600000&to=1670886000000) with the top 50% of contributions coming from IBM, DaoCloud, Google, NEC Corporation, VMWare, and Red Hat. We'd like to give a shout-out to all of the independent contributors who feature third on this ranking in terms of number of contributions
+* Our group also strives to be a positive example when it comes to Diversity, Equity, and Inclusion, where we ensure we have SIG leads from some of our biggest membership bases (India, China), alongside women in technical leadership roles (co-chairs, tech leads)
+
+**Are there ways end users/companies can contribute that they currently are not? If one of those ways is more full time support, what would they work on and why?**
+
+* (to add)
+
+***Membership***
+
+* Primary Slack channel member count: 2,264
+* Primary mailing list member count: 576
+* Primary meeting attendee count (estimated, if needed): 12
+* Primary meeting participant count (estimated, if needed): 6
+* Unique reviewers for SIG-owned packages: xx
+* Unique approvers for SIG-owned packages: xx
+
+***Subprojects***
+
+New in 2022:
+
+    localization
+
+Continuing:
+
+    kubernetes-blog
+    reference-docs
+    website
+
+***Operational***
+
+Operational tasks in sig-governance.md:
+
+- [ ] README.md reviewed for accuracy and updated if needed
+- [ ] CONTRIBUTING.md reviewed for accuracy and updated if needed (or created if missing and your contributor steps and experience are different or more in-depth than the documentation listed in the general contributor guide and devel folder.)
+- [ ] Subprojects list and linked OWNERS files in sigs.yaml reviewed for accuracy and updated if needed
+- [ ] SIG leaders (chairs, tech leads, and subproject owners) in sigs.yaml are accurate and active, and updated if needed
+- [ ] Meeting notes and recordings for 2022 are linked from README.md and updated/uploaded if needed
+- [ ] Did you have community-wide updates in 2022 (e.g. community meetings, KubeCon, or kubernetes-dev@ emails)? Links to email, slides, or recordings:
+    * [Localization Subproject Launch to SIG Docs](https://groups.google.com/g/kubernetes-sig-docs/c/DUFVEi-9tWc/m/ji2vPwBEBQAJ)
+    * (I cannot find our KubeCon NA 2022 talk but lets please list it here)
```

Comment:
They haven't uploaded the whole video, unfortunately, and only Natali's stream exists because we had a hybrid presentation. Although, on second thoughts, we can provide the link to our session's sched. WDYT?

Reply:
Should we also link the Office Hours episode that I did earlier last year since that is technically a community meeting?

https://www.youtube.com/live/KJMka1ZzVUQ?feature=share => This is the link.

Reply:
Yes to both!